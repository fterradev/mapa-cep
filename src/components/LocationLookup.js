import React, { PureComponent } from 'react';
import { Segment, Form, Button } from 'semantic-ui-react';

class LocationLookup extends PureComponent {
  render() {
    return (
      <div>
        <Segment>
          <Form>
            <Form.Group>
              <Form.Input
                inline
                label="CEP"
                type="text"
              />
              <Button>
                Buscar
              </Button>
            </Form.Group>
          </Form>
        </Segment>
      </div>
    );
  }
}

export default LocationLookup;
