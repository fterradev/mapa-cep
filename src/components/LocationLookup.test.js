import React from 'react';
import { shallow } from 'enzyme';
import LocationLookup from './LocationLookup';

describe('Wallet', () => {
  const locationLookup = shallow(<LocationLookup />);

  it('renders properly', () => {
    expect(locationLookup).toMatchSnapshot();
  });
});
