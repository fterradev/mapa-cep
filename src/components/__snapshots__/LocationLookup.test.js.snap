// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wallet renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LocationLookup />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Segment>
        <Form
                as="form"
        >
                <FormGroup>
                        <FormInput
                                as={[Function]}
                                control={[Function]}
                                inline={true}
                                label="CEP"
                                type="text"
                        />
                        <Button
                                as="button"
                                role="button"
                        >
                                Buscar
                        </Button>
                </FormGroup>
        </Form>
</Segment>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <Form
          as="form"
>
          <FormGroup>
                    <FormInput
                              as={[Function]}
                              control={[Function]}
                              inline={true}
                              label="CEP"
                              type="text"
                    />
                    <Button
                              as="button"
                              role="button"
                    >
                              Buscar
                    </Button>
          </FormGroup>
</Form>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "as": "form",
          "children": <FormGroup>
            <FormInput
                        as={[Function]}
                        control={[Function]}
                        inline={true}
                        label="CEP"
                        type="text"
            />
            <Button
                        as="button"
                        role="button"
            >
                        Buscar
            </Button>
</FormGroup>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <FormInput
                as={[Function]}
                control={[Function]}
                inline={true}
                label="CEP"
                type="text"
/>,
              <Button
                as="button"
                role="button"
>
                Buscar
</Button>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "as": [Function],
                "control": [Function],
                "inline": true,
                "label": "CEP",
                "type": "text",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "as": "button",
                "children": "Buscar",
                "role": "button",
              },
              "ref": null,
              "rendered": "Buscar",
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Segment>
          <Form
                    as="form"
          >
                    <FormGroup>
                              <FormInput
                                        as={[Function]}
                                        control={[Function]}
                                        inline={true}
                                        label="CEP"
                                        type="text"
                              />
                              <Button
                                        as="button"
                                        role="button"
                              >
                                        Buscar
                              </Button>
                    </FormGroup>
          </Form>
</Segment>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <Form
            as="form"
>
            <FormGroup>
                        <FormInput
                                    as={[Function]}
                                    control={[Function]}
                                    inline={true}
                                    label="CEP"
                                    type="text"
                        />
                        <Button
                                    as="button"
                                    role="button"
                        >
                                    Buscar
                        </Button>
            </FormGroup>
</Form>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "as": "form",
            "children": <FormGroup>
              <FormInput
                            as={[Function]}
                            control={[Function]}
                            inline={true}
                            label="CEP"
                            type="text"
              />
              <Button
                            as="button"
                            role="button"
              >
                            Buscar
              </Button>
</FormGroup>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <FormInput
                  as={[Function]}
                  control={[Function]}
                  inline={true}
                  label="CEP"
                  type="text"
/>,
                <Button
                  as="button"
                  role="button"
>
                  Buscar
</Button>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "as": [Function],
                  "control": [Function],
                  "inline": true,
                  "label": "CEP",
                  "type": "text",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "as": "button",
                  "children": "Buscar",
                  "role": "button",
                },
                "ref": null,
                "rendered": "Buscar",
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
